OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/ChildControllerTests.cs

```diff
@@ -33,7 +35,9 @@ public class ChildControllerTests
         public void Setup()
         {
             service = new Mock<IChildService>();
-            controller = new ChildController(service.Object);
+            providerService = new Mock<IProviderService>();
```

Comment:
`providerService` and `providerAdminService` might be done local method variables, not class fields. Encapsulation might be improved